### PR TITLE
Updated project settings, Fixed issues for dynamic framework

### PR DESCRIPTION
**Pull request by amosavian**
This pull request addressed framework issues discussed in https://github.com/abbeycode/UnzipKit/issues/48 allows project to be used for manual dynamic linking.

### CODE REVIEW

**Comment by abbeycode** [comment]
Can you please merge in upstream changes that fixed the build?

File: Resources/UnrarKit-Info.plist [hunk 1]
```diff
@@ -15,10 +15,10 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.8-beta</string>
```
Comment (abbeycode) [inline Resources/UnrarKit-Info.plist]: I'll update it out of beta after we merge this request. I've got a scripted process for doing so.

File: UnrarKit.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1011,9 +1011,10 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0700;
-				LastUpgradeCheck = 0700;
```
Comment (abbeycode) [inline UnrarKit.xcodeproj/project.pbxproj]: Were any of the changes to this file required for the framework to build?

File: UnrarKit.xcodeproj/xcshareddata/xcschemes/UnrarKit iOS.xcscheme [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0700"
+   LastUpgradeVersion = "0820"
```
Comment (abbeycode) [inline UnrarKit.xcodeproj/xcshareddata/xcschemes/UnrarKit iOS.xcscheme]: Please revert this

File: UnrarKit.xcodeproj/xcshareddata/xcschemes/UnrarKit.xcscheme [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0700"
```
Comment (abbeycode) [inline UnrarKit.xcodeproj/xcshareddata/xcschemes/UnrarKit.xcscheme]: Please revert this

**Comment by abbeycode** [comment]
Thanks for the contribution - as far as I can tell though, only the additional import is required though, right? That's basically what all of my individual comments were getting at.

File: UnrarKit.xcodeproj/project.pbxproj (hunk 2 above)
Comment (amosavian) [inline UnrarKit.xcodeproj/project.pbxproj]: I will only make a warning silent. Nothing special.

File: UnrarKit.xcodeproj/xcshareddata/xcschemes/UnrarKit iOS.xcscheme (hunk 3 above)
Comment (amosavian) [inline UnrarKit.xcodeproj/xcshareddata/xcschemes/UnrarKit iOS.xcscheme]: I will only make a warning silent. Nothing special.

File: UnrarKit.xcodeproj/xcshareddata/xcschemes/UnrarKit.xcscheme (hunk 4 above)
Comment (amosavian) [inline UnrarKit.xcodeproj/xcshareddata/xcschemes/UnrarKit.xcscheme]: ditto

**Comment by amosavian** [comment]
In general it adds `TargetConditionals.h` to umbrella header, remove beta tag from version, and change LastUpgrade to Xcode 8.2 to silence a warning

**Comment by abbeycode** [comment]
Right, so for the beta tag and LastUpgrade flag, those are things that I'd rather take care of myself.

For removing "beta", I will do that after merging this PR. For the version upgrade, I need to check over all the upgraded settings. It shouldn't take me long, but I can't promise a timeframe. I'll try to look at it soon.

**Comment by amosavian** [comment]
Consequently only `TargetConditionals.h` to umbrella header left. I appreciate if you add it too.

**Comment by amosavian** [comment]
Also you can switch to `Foundation` from `UIKit` and `Cocoa` if the lib didn't used anything special to these frameworks. Makes it compatible with even future Apple OSes. Which means `TargetConditionals.h` not necessary anymore.

**Comment by abbeycode** [comment]
Good call on using Foundation. I made that switch, and pushed a new build to CocoaPods (everything is at 2.8 now, no longer beta).

**Pull request closed without merging**